platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -611,6 +619,38 @@ export default class HangingProtocolService extends PubSubService {
       return defaultReturn;
     }
 
+    const originalProtocol = this._originalProtocol;
+    let originalProtocolStage;
+    if (!(originalProtocol instanceof Function)) {
+      originalProtocolStage = originalProtocol.stages[this.stageIndex];
+    }
+
+    function updateDisplaySetInstanceUIDs(
```

Comment:
Please define this outside the function - you aren't taking accessing any local variables AFAIK, so you can just call it as either:
HangingProtocolService.updateDisplaySetInstanceUIDs
or this.update....